core/chains/evm/log/broadcaster.go

```diff
@@ -188,7 +188,7 @@ func (b *broadcaster) Start(context.Context) error {
 
 // ReplayFromBlock implements the Broadcaster interface.
 func (b *broadcaster) ReplayFromBlock(number int64, forceBroadcast bool) {
-	b.logger.Infof("Replay requested from block number: %v", number)
+	b.logger.Infow("Replay requested", "block number", number, "force", forceBroadcast)
```

Comment:
It looks like we use the key `"blockNum"` a lot? :shrug: 
```suggestion
	b.logger.Infow("Replay requested", "blockNum", number, "force", forceBroadcast)
```